cfbs/commands.py

```diff
@@ -201,15 +202,96 @@ def local_module_copy(module, counter, max_length):
         f"{counter:03d} {pad_right(name, max_length)} @ local                                    (Copied)"
     )
 
+def _get_path_from_url(url):
+    if not url.startswith(("https://", "ssh://", "git://")):
+        if ("://" in url):
+            return user_error("Unsupported URL protocol in '%s'" % url)
+        else:
+            # It's a path already, just remove trailing slashes (if any).
+            return url.strip("/")
```

Comment:
This would also strip a leading slash. Consider using rstrip, adding an assert / error message or at least updating the comment.